galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.openstack.manager/src/main/java/dev/galasa/openstack/manager/internal/properties/OpenStackFloatingIPPool.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.openstack.manager.internal.properties;
+
+import dev.galasa.framework.spi.ConfigurationPropertyStoreException;
+import dev.galasa.framework.spi.cps.CpsProperties;
+import dev.galasa.openstack.manager.OpenstackManagerException;
+
+/**
+ * OpenStack Floating IP Pool
+ * <p>
```

Comment:
Don't do this `<p>` stuff. Just leave a blank line. 
It has the same effect, and it's easier to read the code.

Reply:
Okay sure - I had replicated the same format as the other Javadoc comments in this bundle here - will fix in a separate PR as I'd like this to build overnight and run in tomorrow's regression run.